site/docs/components/button/accessibility.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Button
+layout: DetailComponent
+sidebar:
+  label: Accessibility
+data:
+  descriptionRef:
+    $ref: ./#/data/description
+---
+
+### Best practises
+
+#### When a Button has both an icon and a label
+
+- Pass `aria-hidden=”true”` to the icon so that its label is not announced by screen readers.
+
+#### When a Button has an icon only and no label
+
+- Pass an `aria-label` to Button that describes its purpose.
+
+### Keyboard interactions
+
+| Key                                  | Function                                                                                                                                                                                                                                                                |
+| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <kbd>Tab</kbd>                       | - If focus is above the button, Tab key press moves focus onto the button.<br/>- If the button is disabled and is set to receive focus, receives focus.<br/>- If the button has focus, Tab moves focus out of the button, to next focusable component in the tab order. |
```

Comment:
```suggestion
| <kbd>Tab</kbd>                       | - If focus is above the Button, Tab key press moves focus onto the Button.<br/>- If the Button is disabled and is set to receive focus, receives focus.<br/>- If the Button has focus, Tab moves focus out of the Button, to next focusable component in the tab order. |
```

Reply:
I'm happy to make these changes but this page for now is just an example that I've copied from Figma so we could see the layouts in action. I think we'll need a proper copy review as we add the component docs into the site. What do you reckon @james-nash @shey-v ?

Reply:
awesome, I guess it will be checked anyway by content :)

Reply:
Agreed. This PR is just about the technical mechanism of rendering MDX content split across different files into the tabs of the component page layout.

Migrating existing content into the page layout (once it's all done) will be a separate exercise - see #1044